Added seminars

JeffCarver opened the pull request:
## Description
<!--- Describe your changes.  If it fixes an open issue, please link to the issue here. -->


## Checklist:
<!---Check these off after you create the PR --->
- [x] I have previewed changes locally or with CircleCI (runs when PR is created)
- [ ] I have completed any content reviews, such as getting input from relevant working groups.  If no, please note this and wait to post the PR to the #website channel until the content has been settled.  


When you are ready for a technical review/merge, post the for the link for the PR in the US-RSE Slack (#website) to ask for reviewers.

<!---Ask questions if needed in the #website channel of US-RSE Slack --->

lmilechin commented on pages/wg/education_training.md:
```diff
@@ -17,6 +17,15 @@ The US-RSE Education & Training working group has produced the following documen
+    * [Abstract](https://us-rse.org/events/2022/2022-10-education-training-seminar/)
+    * [Video](https://youtu.be/Fdex2rhw0Zo)
+* Setting up a Scientific Software Project using GitLab -- From Zero to Hero
+    * [Abstract] https://us-rse.org/events/2023/2023-03-education-training-tutorial/
```
There is an extra space between [Abstract] and the URL

JeffCarver commented:
Fixed typo with abstract. Updated preview: https://output.circle-artifacts.com/output/job/cff0959a-1a2f-46f3-a4f4-54d6d345b14c/artifacts/0/usrse.github.io/wg/education_training/index.html
